### PR TITLE
Configuration for storage location tracking to use movements

### PR DESCRIPTION
**Pull request by kehh**
- http://docs.collectiveaccess.org/wiki/Location_tracking

### CODE REVIEW

**Comment by kehh** [comment]
I just discovered this change which also needed pushing

File: multipart_id_numbering.conf [hunk 1]
```diff
@@ -0,0 +1,311 @@
+#
```
Comment (leftclickben) [inline multipart_id_numbering.conf:1]: Maybe just a comment that says "see `multipart_id_numbering.conf`" in the base configuration directory in `providence`?

File: multipart_id_numbering.conf [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+			}
+		}
+	},
+# ----------------------------------------------------
+#
+#	Below is an example of a default format for objects that will format
+#	id numbers in the format <year>.<code>.<number>; for example: 2009.ORG.10
+# 	where <number> is automatically assigned
+#
+#	ca_objects = {
```
Comment (leftclickben) [inline multipart_id_numbering.conf]: Please remove this commented-out code

File: multipart_id_numbering.conf (hunk 1 above)
Comment (kehh) [inline multipart_id_numbering.conf:1]: I put a link to http://docs.collectiveaccess.org/wiki/Multipart_id_numbering.conf instead

**Comment by kehh** [comment]
ok all done

**Pull request merged**